### ISSUE TITLE
Limit size of backup directory

### ISSUE THREAD

**Issue by woutgg** [marchyman/GeoTag#14]
First of all, thanks for this useful program!

I often tag my photos just before importing them into LR and I like how the originals are backed up.

The thing is, that directory grows over time while tagging more and more photos and of course I can empty it manually each time (assuming that I remember to do so). I think however, that it would be nice if it could be pruned automatically based on file age and/or directory size.

**Comment by marchyman** [comment]
Nice idea.   Thank you.  Let me think on how I might do that.  I'm inclined to not delete the backups, but instead put up a warning.  Or something like that.

**Comment by woutgg** [comment]
I can understand why you're hesitant to delete backups, and it should definitely only happen in a way users expect.

Perhaps it is enough to disable such functionality by default so it only happens after enabling it in the preferences (this also ensures 'backwards compatible' behaviour).
Or, if you feel that is not safe enough, the operation could be put behind a dialog (or menu item, or something else), forcing the user to actively decide to clean up the backups each time.

For my workflow, auto-cleaning based on file age would probably work best, as I only use the backups to bridge the time between tagging and importing. As soon as I see them in LR I know everything is in order and I don't need those backups anymore.

**Comment by marchyman** [comment]
I added a reminder to clean your backup/save folder.  The reminder is triggered if the backup/save folder contains more than 500 MB or if any file was added to the folder more than 7 days prior.  I know it's not what you really asked for, but it is what I'm comfortable doing right now.

![Screen Shot 2020-02-08 at 1 08 13 PM](https://user-images.githubusercontent.com/784627/74092148-098bb000-4a75-11ea-9cc1-80ea70763c52.png)
